src/components/organisms/HomeContent.tsx

```diff
@@ -72,15 +73,18 @@ export default function HomeContent(): ReactElement {
     ),
     image: (
       <>
-        <div onClick={() => setIsOpen(true)}>
-          <Img fluid={ctdBenefitsImage} />
+        <div className={styles.imagePreview} onClick={() => setIsOpen(true)}>
+          <img src={ctdBenefitsImage} alt="Compute-to-Data and it's benefits" />
```

Comment:
I just noticed: it actually needs to say : "Compute-to-Data and **its** benefits"